modules/price/src/Plugin/Field/FieldFormatter/PriceCalculatedFormatter.php

```diff
@@ -79,8 +86,10 @@ class PriceCalculatedFormatter extends PriceDefaultFormatter implements Containe
    *   The current user.
    */
   public function __construct($plugin_id, $plugin_definition, FieldDefinitionInterface $field_definition, array $settings, $label, $view_mode, array $third_party_settings, EntityTypeManagerInterface $entity_type_manager, NumberFormatterFactoryInterface $number_formatter_factory, ChainPriceResolverInterface $chain_price_resolver, StoreContextInterface $store_context, AccountInterface $current_user) {
-    parent::__construct($plugin_id, $plugin_definition, $field_definition, $settings, $label, $view_mode, $third_party_settings, $entity_type_manager, $number_formatter_factory, $store_context);
+    parent::__construct($plugin_id, $plugin_definition, $field_definition, $settings, $label, $view_mode, $third_party_settings, $entity_type_manager, $number_formatter_factory);
+
     $this->chainPriceResolver = $chain_price_resolver;
+    $this->storeContext = $store_context;
```

Comment:
This isn't being injected anymore. The constructor lists it but you did not override the parent `create` method.